app/controllers/steps/screener/parent_controller.rb

```diff
@@ -0,0 +1,15 @@
+module Steps
+  module Screener
+    class ParentController < Steps::ScreenerStepController
+      def edit
+        @form_object = ParentForm.new(
```

Comment:
You want to use `#build` here instead of `#new`, otherwise the value of the attribute is not retrieved for rendering.
In some forms we use #new but then we also need to pass the attribute(s) to set the values.